configure.ac

```diff
@@ -26,11 +26,6 @@ AC_TYPE_UINT8_T
 AC_CHECK_FUNCS([malloc memmove memset strchr strtol])
 
 AX_CFLAGS_WARN_ALL
-AX_COMPILER_FLAGS([-Wpedantic -Wall -Wextra -Wcast-align -Wcast-qual\
```

Comment:
No, those two are not equivalent at all. See the documentation on AX_CFLAGS_WARN_ALL.